docs/advanced-usage/access-control/kerberos/index.md

```diff
@@ -47,10 +47,24 @@ $ reana-client secrets-add --env CERN_USER=johndoe \
 
 ## Setting Kerberos requirement
 
-Set `kerberos: true` for the steps in need in the workflow specification.
-Please note that step's docker image (e.g `environment: 'cern/slc6-base'`)
-should have Kerberos client installed and you have to for the Kerberos
-authentication to work.
+If the workflow engine you are using needs Kerberos to parse and validate the
+workflow specification, then you can enable it globally for the whole workflow
+in the `reana.yaml` file. For example, this is needed if you want to access
+restricted resources directly from your `Snakefile`. This will also set up
+Kerberos for each step of your workflow.
+
+```yaml hl_lines="4"
+workflow:
+  type: snakemake
+  resources:
+    kerberos: true
+  file: workflow/snakemake/Snakefile
+```
+
+Otherwise, you can set `kerberos: true` for the steps that need it in the
+workflow specification. Please note that the steps' docker images (e.g
+`environment: 'cern/slc6-base'`) should have the Kerberos client installed for
```

Comment:
Actually this is not true. The Kerberos authentication is done via an init sidecar container, so the job images do not have to have anything Kerberos-related built-in... It'll work anyway. I know you have copied the existing phrase that we had before, but let's fix it as part of this PR. You can simply delete the entire phrase ("Please note...").

Reply:
> It would be actually good if we support it for _all_ workfow engines... It'll create a nice coherency with how CVMFS is supported. So I'd vote for keeping docs generic and adding PRs to other engines progressively...

Related issues:
- https://github.com/reanahub/reana-workflow-engine-cwl/issues/228
- https://github.com/reanahub/reana-workflow-engine-serial/issues/167
- https://github.com/reanahub/reana-workflow-engine-yadage/issues/231